bigquery/google/cloud/bigquery/job.py

```diff
@@ -702,6 +707,25 @@ def __init__(self, job_type):
         self._job_type = job_type
         self._properties = {job_type: {}}
 
+    @property
+    def labels(self):
+        """Dict[str, str]: Labels for the job.
+
+        This method always returns a dict. To change a job's labels,
+        modify the dict, then call ``Client.update_job``. To delete a
```

Comment:
I'm trying to use this feature to update a job's labels from the Python client, but there does not seem to be a method `Client.update_job`. Is this feature incomplete?

Reply:
@rarredon A very late answer, but if it still helps - there was a [bug](https://github.com/googleapis/python-bigquery/issues/469) opened that tracks this.

I doesn't seem that jobs can be modified, though, since the [jobs API](https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs) does not support PATCH-ing them (unless there is an issue with the docs themselves).

BTW, for future reference, its best to ask questions in the new repository's [issue tracker](https://github.com/googleapis/python-bigquery/issues). This is an old PR in the old repository that most library maintainers rarely monitor (if at all).

Reply:
Thanks @plamut! I managed to find a workaround. I will add a comment to the issue you linked.